sintTorqueController:
  type: arm_controllers/sintTorqueController
  joints:
    - elfin_joint1
    - elfin_joint2
    - elfin_joint3
    - elfin_joint4
    - elfin_joint5
    - elfin_joint6
  gains:
    alpha: 0.1
    elfin_joint1:
      pid: {p: 8, i: 0.0, d: 0.1, i_clamp: 50, antiwindup: true}
    elfin_joint2: 
      pid: {p: 8, i: 0.0, d: 0.1, i_clamp: 50, antiwindup: true}
    elfin_joint3:
      pid: {p: 8, i: 0.0, d: 0.1, i_clamp: 50, antiwindup: true}
    elfin_joint4: 
      pid: {p: 8, i: 0.0, d: 0.1, i_clamp: 50, antiwindup: true}
    elfin_joint5: 
      pid: {p: 8, i: 0.0, d: 0.1, i_clamp: 50, antiwindup: true}
    elfin_joint6: 
      pid: {p: 8, i: 0.0, d: 0.1, i_clamp: 50, antiwindup: true}
  root_link: world
  tip_link: elfin_link6
  publish_rate: 500